README.md

```diff
@@ -75,7 +75,7 @@ To build a binary, run `go build && ./server -h`. See [@tehbilly](https://github
 - Add project to package managers
   - [x] npm
   - [x] composer
-  - [ ] aur
+  - [x] aur
```

Comment:
@triforce, I think you have an outdated fork. At least, this `README.md` content is. No worries though, I merged in your PR.